test/test_01_guess_cmake.py

```diff
@@ -0,0 +1,33 @@
+
+import time
+import pytest
+import config
+import subprocess
+
+def ExecutablesOfBuffer(path):
+    return "guess_executable_cmake#ExecutablesOfBuffer('" + path + "')"
+
+def test_cmake_completion(eng):
+    subprocess.run(["cmake", "src", "-B", "src/build"])
```

Comment:
I suggest moving this command to `prerequisites.sh`. This would potentially allow fine-tuning the tests. Like skipping if there is no cmake in the system. For example, if we install cmake in Linux and there is no suitable version in Darwin readily available, some tests could be skipped on Darwin. It would be enough to test on just Linux.